CONTRIBUTING.rst

```diff
@@ -141,13 +141,17 @@ How to set up a development environment
 ---------------------------------------
 
 For working on gcovr, you will need a supported version of Python 3,
-and GCC version 5. Other GCC versions are supported by gcovr,
+and GCC version 5, 6 or 8. Other GCC versions are supported by gcovr,
 but will cause spurious test failures.
 
 -   (Optional) Fork the project on GitHub.
 
 -   Clone the git repository.
 
+-   (Optional) If your compiler executable is gcc have to create a
+    symbolic link or copy with the version of the GCC. E.g. if it is GCC
+    version 8 you have to create a gcc-8.
```

Comment:
It is not clear to me what this step is instructing. Presumably the goal is that there's an executable called `gcc-8` in the PATH, correct? I have no idea how to do that on Windows, but we could at least add simple Linux/MacOS instructions. Something like:

> If you're on Linux or MacOS, you can check the GCC version with `gcc --version`. If the output says version 8, you should also be able to run `gcc-8 --version`. Your Linux distribution should have set all of this up already.
>
> If you don't have an alias like `gcc-8`, perform the following steps to create an alias for `gcc`:
>
> 1. Create a directory somewhere, e.g. in your home directory: `mkdir ~/bin`
> 2. Create a symlink in that directory which points to GCC: `ln -s $(which gcc) ~/bin/gcc-8`
> 3. Add this directory to your PATH: `export PATH="$HOME/bin:$PATH"`
> 4. Re-test `gcc-8 --version` to ensure everything worked.
> 5. Create additional symlinks for `g++` :arrow_right: `g++-8` and `gcov` :arrow_right: `gcov-8`.

Alternatively, we could just promote the Docker-based workflow as the default environment, and essentially deprecate running the tests on the host. Anecdotally there can be some issues with getting Docker to run on Windows, but once that's out of the way the rest is much simpler.

Reply:
Great instruction to add symliks. This should also work in the MSYS environment under Windows. I'll use this instruction.

Reply:
Probably, this issue is a mingw/cygwin/msys2 only issue. This seems to be a problem only for a few tests. I'm wondering why the other tests work
The symlink instructions are not "complete" since in a normal linux pc, you have all gcc related binaries with the "dash version" naming, including:
* gcc
* gcc-nm
* gcc-ar
* g++
* gcov
* gcov-tool
and probably many I have forgotten...

so, is there any possibility to fix this in the two tests that are failing?